config/routes.rb

```diff
@@ -22,9 +22,11 @@
   get "/accessibility_statement", to: "static_pages#accessibility_statement", as: :accessibility_statement
 
   constraints slug: %r{#{PageSequence::SLUGS.join("|")}} do
-    resources :claims, only: [:new, :create, :show, :update], param: :slug, path: "/claim"
+    resources :claims, only: [:new, :show, :update], param: :slug, path: "/claim"
   end
 
+  resources :claims, only: [:create], params: {slug: "qts-year"}, path: "/claim"
```

Comment:
Can we write this as singular route definition, rather than a resource that is only a single action? I think it would be easier to follow written like that. Also can we parameterise the first slug via `PageSequence` rather than hard coding?

Reply:
Good call 👍